test/extended/cluster/cl.go

```diff
@@ -81,30 +81,34 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 				e2e.Logf("%d/%d : Created new namespace: %v", j+1, p.Number, nsName)
 				namespaces = append(namespaces, nsName)
 
-				// Create templates as defined
-				for _, template := range p.Templates {
-					var allArgs []string
-					templateFile := mkPath(template.File)
-					e2e.Logf("We're loading file %v: ", templateFile)
-					templateObj, err := testutil.GetTemplateFixture(templateFile)
-					if err != nil {
-						e2e.Failf("Cant read template config file. Error: %v", err)
+				// Create config maps
+				if p.Configmaps != nil {
+					// Config maps defined
+					for k, v := range p.Configmaps {
+						args, err := getConfigMapArgs(k, v)
+						o.Expect(err).NotTo(o.HaveOccurred())
+						err = oc.SetNamespace(nsName).Run("create").Args(args...).Execute()
+						o.Expect(err).NotTo(o.HaveOccurred())
 					}
-					allArgs = append(allArgs, templateObj.Name)
+				}
 
-					if template.Parameters == nil {
-						e2e.Logf("Template environment variables will not be modified.")
-					} else {
-						params := convertVariablesToString(template.Parameters)
-						allArgs = append(allArgs, params...)
+				// Create secrets
+				if p.Secrets != nil {
+					// Secrets defined
+					for k, v := range p.Secrets {
```

Comment:
This block looks really similar to the configmap version L87-91 😄 
Can we make it more generic too?